openapi/src/main/resources/templates/openworld-sdk/fieldConstraintViolation.mustache

```diff
@@ -0,0 +1,7 @@
+package {{packageName}}.models.exception
+
+data class FieldConstraintViolation(
+    val fieldName: String,
```

Comment:
Should we rename the properties to `name` and `path`?

Reply:
I presume the "field" part suggests where the problem is coming exactly from. Since, to the user, it is not clear what the constraint is set on.

Reply:
Rethinking this.. After documenting them, it must be more clear to the user. I will be renaming them as you said